sites/svelte.dev/vite.config.js

```diff
@@ -36,6 +20,50 @@ function raw(ext) {
 	};
 }
 
+/** @returns {import('vite').Plugin} */
+function minified_raw_plugin() {
+	const prefix = 'minified-raw:';
```

Comment:
i've been starting virtual imports with `\0virtual:` in sveltekit as I think it's clearer than just `\0`, which you need some prior knowledge of
```suggestion
	const prefix = 'virtual:minified-raw-';
```